guides/v2.2/architecture/archi_perspectives/components/modules/mod_depend.md

```diff
@@ -31,13 +31,45 @@ A module with a *hard dependency* on another module cannot function without the
 * Deserializes an object declared in another module.
 * Uses or modifies the database tables used by another module.
 
+Module hard dependency defines in the `require` section of `app/code/<Vendor>/<Module>/composer.json` file. For example, :
```

Comment:
Please adjust this part to make it more clear
```suggestion
Module hard dependency is defined in the `require` section of `app/code/<Vendor>/<Module>/composer.json` file. For example, :
```

Reply:
Do not make this passive. Is defined is not an improvement.